### ISSUE TITLE
placeholders

### ISSUE THREAD

**Issue by RaydenKonig** [VolmitSoftware/Iris#271]
I think it would be great to implement IRIS placeholders
that give information such as the name of the biome in which the user is, or region
etc
I do not see it very difficult to implement since that same information is in the editor mode marker

I for example want this implementation to be able to detect when a user enters a biome for the first time to give him an achievement and execute commands with a plugin

but i need the placeholders

**Comment by RaydenKonig** [comment]
It would also be great to be able to add events in the api of when a user enters or leaves a region
this would give server developers a lot of possibilities to execute commands when entering a region
using plugins like "conditional events"

I, for example, want to use it to detect when a player enters a region and when they leave it.

to be able to use a musical complement and play a respective soundtrack depending on the biome or region in which the player is located

I don't know if this would be very difficult to implement since I don't know about plugin programming.

**Comment by AZFire1** [comment]
I'd appreciate placeholders as well for Iris, especially for biomes. I'd love to add them to my scoreboard as some extra info for players :)

**Comment by ghost** [comment]
https://www.spigotmc.org/resources/placeholderapi.6245/

**Comment by AZFire1** [comment]
@ArMiN231 yes, placeholders that hook into PAPI which could be used in other plugins.

**Comment by ghost** [comment]
It’s free and most user system for placeholders. I posted that link for Coco to remember that thing when working on it.
